package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mazzaroth-js",
-  "version": "0.2.6",
+  "version": "0.2.7",
```

Comment:
I don't think you need to update the version number here.  This gets updated by CirclCI currently when a release branch is created.

Reply:
Value restored. Thanks.